plugin-worldclock/lxqtworldclock.cpp

```diff
@@ -481,7 +481,7 @@ void LXQtWorldClock::updatePopupContent()
 bool LXQtWorldClock::formatHasTimeZone(QString format)
 {
     format.replace(QRegExp(QLatin1String("'[^']*'")), QString());
-    return format.toLower().contains(QLatin1String("t"));
+    return format.contains(QLatin1Char('t'));
```

Comment:
A typo? I think it should be as this:
```c++
return format.contains(QLatin1Char('t'), Qt::CaseInsensitive);
```
When time zone is shown, `mFormat` can contain strings like `TTTTT` (→ `LXQtWorldClock::settingsChanged`) and `formatHasTimeZone` is called on it.

Reply:
> A typo? I think it should be as this:

And you are thinking right. A mental typo.

Reply:
Being lazy in typing, I sometimes copy-paste codes, change them and forget to add necessary parts ;) Hence thinking about typo.

Reply:
The expressions case sensitive / case insensitive can (don't know why) can confuse Portuguese speakers. It's easy to think one thing them type the opposite.
Another example, is push/pull, although the explanation is far easier/obvious.